src/java.base/share/classes/java/lang/StackTraceElement.java

```diff
@@ -558,12 +553,20 @@ static StackTraceElement of(StackFrameInfo sfi) {
         return ste;
     }
 
+    static StackTraceElement[] of(StackTraceElement[] stackTrace) {
```

Comment:
Is it intentional that this method is returning a `StackTraceElement[]` instead of `void`. The current implementation just returns back the passed array after operating on each element. Are callers mandated to use the returned array for subsequent operations instead of the passed array?

Reply:
It's just for convenience of use.